src/lib/dom.iterable.d.ts

```diff
@@ -5,9 +5,17 @@ interface DOMTokenList {
 }
 
 interface NodeList {
+    keys(): IterableIterator<number>;
+    values(): IterableIterator<[number, Node]>;
+    entries(): IterableIterator<Node>;
+    forEach(callbackfn: (value: Node, index: number, listObj: NodeList) => void, thisArg?: any): void;

```

Comment:
would also appreciate it if you added the comments i sugested. this way users get nice help message to read instead of going to MDN.